circuit/src/execution-delivery/src/exec_composer/mod.rs

```diff
@@ -504,6 +516,37 @@ mod tests {
         hex_literal::hex!["d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"].into()
     }
 
+    fn make_registry_contract_out_of_wat<T: Config>(
```

Comment:
Possible to re-use that function from other files where  both `make_compose_out_of_raw_wat_code` + `RegistryContract::from_compose` are used? pretty sure it's the case this logic has been implemented in some other tests too

Reply:
As per offline discussion, resolving this comment.